apps/dashboard/src/components/editor/editor.js

```diff
@@ -108,23 +108,34 @@ const Editor = ( { project } ) => {
 
 	const settings = useMemo( () => {
 		if ( ! confirmationPage ) {
-			return baseSettings;
+			return editorSettings;
 		}
 
-		return tap( cloneDeep( baseSettings ), ( { editor, iso } ) => {
+		return tap( cloneDeep( editorSettings ), ( { editor, iso } ) => {
```

Comment:
I wonder, should we move this to useEditorContent, or can we even maybe incorporate it into `getEditorSettings`?
It's the same store so you do have access to `confirmationPage` there, the trick would be to not create a new object every time that selector is run as that may cause unnecessary reloads.

You'd need some sort of memoization to solve that - I think `reselect` is the most straight forward approach historically, and we've actually used it within the old frontend too so I think it's a safe one to install.

Reply:
Thanks for the suggestion and for pointing out the `reselect` lib.
I tried to achieve what you suggested with [e8d7994](https://github.com/Automattic/crowdsignal-ui/pull/317/commits/e8d799460c3381c158b1767b993f05af29bde407).
Let me know if you thought something different or if something is missing.